infra/scripts/validate-version-consistency.sh

```diff
@@ -123,7 +125,7 @@ for i in "${files_to_validate_version[@]}"; do
   echo "Testing whether versions are correctly set within file: $FILE_PATH"
   ACTUAL_OCCURRENCES=$(grep -c -P "\bv?$VERSION\b" "$FILE_PATH" || true)
 
-  if [ "${ACTUAL_OCCURRENCES}" -eq "${EXPECTED_OCCURRENCES}" ]; then
+  if [ "${ACTUAL_OCCURRENCES}" -ge "${EXPECTED_OCCURRENCES}" ]; then
```

Comment:
Actual occurrences is greater than that expected occurrences when both pom.xml and tag versions the same. This is not a issue when pom.xml version is SNAPSHOT, but when its not in SNAPSHOT form, lint version scripts will flag extra occurrences and fail.